frappe/core/doctype/file/utils.py

```diff
@@ -269,10 +275,16 @@ def _save_file(match):
 
 	if content and isinstance(content, str):
 		content = re.sub(r'<img[^>]*src\s*=\s*["\'](?=data:)(.*?)["\']', _save_file, content)
+		content = content.replace('<img src="">', get_broken_image_replacement())
```

Comment:
* I guess we could just set this directly on BinasciiError instead? more so given _save_file is an inner function 
* Would also show "broken image text" when someone _(however unlikely)_ would add `<img src="">`?